pbin.sh

```diff
@@ -100,13 +106,13 @@ set_defaults() {
 	unset reply
 
 	# default to user@hostname
-	name=${SUDO_USER:-$USER}@${HOSTNAME:-$(hostname)}
+	name=${SUDO_USER:-$USER}
```

Comment:
Ok, I thought it would just be more informative that root and librenms for everything :)  Then you could tell if multiple pastes were made by he same user.